### PR TITLE
add ovs internal-port for pod network interface

### PR DESCRIPTION
**Pull request by hongzhen-ma**
#### What type of this PR
Examples of user facing changes:
- API changes
- Bug fixes

#### Which issue(s) this PR fixes:
Fixes #(issue-number)

### CODE REVIEW

File: pkg/daemon/handler.go [hunk 1]
```diff
@@ -130,7 +132,11 @@ func (csh cniServerHandler) handleAdd(req *restful.Request, resp *restful.Respon
 
 	if strings.HasSuffix(podRequest.Provider, util.OvnProvider) && subnet != "" {
 		klog.Infof("create container interface %s mac %s, ip %s, cidr %s, gw %s", ifName, macAddr, ipAddr, cidr, gw)
-		err := csh.configureNic(podRequest.PodName, podRequest.PodNamespace, podRequest.Provider, podRequest.NetNs, podRequest.ContainerID, ifName, macAddr, ipAddr, gw, ingress, egress, vlanID, podRequest.DeviceID)
+		if nicType == util.VethType {
```
Comment (oilbeater) [inline pkg/daemon/handler.go:135]: when type is empty should also use vethpair

File: pkg/daemon/ovs.go [hunk 2]
```diff
@@ -158,8 +168,20 @@ func configureContainerNic(nicName, ifName string, ipAddr, gateway string, macAd
...
-			return err
+		if nicType == util.VethType {
+			if err = configureNic(ifName, ipAddr, macAddr, mtu); err != nil {
+				return err
+			}
+		} else {
+			if err = configureNic(nicName, ipAddr, macAddr, mtu); err != nil {
+				return err
+			}
+			if err = addStaticEth0Nic(); err != nil {
```
Comment (oilbeater) [inline pkg/daemon/ovs.go:179]: should use `ifName` instead of `eth0`

File: pkg/daemon/ovs.go [hunk 3]
```diff
@@ -140,9 +147,12 @@ func configureContainerNic(nicName, ifName string, ipAddr, gateway string, macAd
 	}
 
 	return ns.WithNetNSPath(netns.Path(), func(_ ns.NetNS) error {
-		if err = netlink.LinkSetName(containerLink, ifName); err != nil {
-			return err
+		if nicType == util.VethType {
```
Comment (oilbeater) [inline pkg/daemon/ovs.go:150]: when empty

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: pkg/daemon/handler.go (hunk 1 above)
Comment (hongzhen-ma) [reply pkg/daemon/handler.go]: Modified.
nicType has been assigned a default value "veth-pair" in kube-ovn-controller.

File: pkg/daemon/ovs.go (hunk 2 above)
Comment (hongzhen-ma) [reply pkg/daemon/ovs.go]: Modified.

File: pkg/daemon/ovs.go (hunk 3 above)
Comment (hongzhen-ma) [reply pkg/daemon/ovs.go]: Modified.

**Pull request merged**